browser/help/topics/v3-ht-annotations.md

```diff
@@ -11,9 +11,11 @@ The gnomAD v3 Hail Table annotations are defined below:
 - **freq_index_dict**: Dictionary keyed by specified label grouping combinations (group: adj/raw, pop: gnomAD inferred global population, sex: sex karyotype), with values describing the corresponding index of each grouping entry in the ‘freq’ array row annotation.
 - **faf_index_dict**: Dictionary keyed by specified label grouping combinations (group: adj/raw, pop: gnomAD inferred global population, sex: sex karyotype), with values describing the corresponding index of each grouping entry in the filtering allele frequency (‘faf’) row annotation.
 - **faf_meta**: Filtering allele frequency metadata. An ordered list containing the frequency aggregation group for each element of the ‘faf’ array row annotation.
-- **VEP version**: VEP version that was run on the callset.
-- **vep_csq_header**: VEP header for VCF export.
-- **dbsnp_version**: dbSNP version used in the callset.
+- **vep_globals**: Information about VEP annotations. 
+  - **vep_version**: VEP version that was run on the callset.
+  - **vep_csq_header**: VEP header for VCF export.
+  - **vep_help**: Output from vep --help.
+  - **vep_config**: VEP configuration to run VEP with [Hail](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.vep).
```

Comment:
I'm not sure if we should add the direct link to the script here (responding to your comment0 given that our code may change after the v4 release (meaning the line number could point someone to the wrong location). maybe we should say something like "File created using command within VEP init shell script in https://github.com/broadinstitute/gnomad_methods/tree/main"

Reply:
yeah, sounds good to me, because it's slightly different from default Hail vep.